### PR TITLE
[CLOUDGA-22732][CLOUDGA-22733][CLOUDGA-22858]Add get and delete PITR Config and poll for the tasks

### PR DESCRIPTION
**Pull request by harshitha-yb**
Add get and delete PITR Config and poll for the tasks

### CODE REVIEW

File: cmd/cluster/pitr-config/pitr_config_ops.go [hunk 1]
```diff
@@ -97,15 +149,35 @@ var createPitrConfigCmd = &cobra.Command{
...
+		pitrConfigId := resp.Data.Info.Id
 
-		fmt.Printf("\nSuccessfully created PITR configuration.\n\n")
+		msg := fmt.Sprintf("The PITR Configuration for namespace %s in cluster %s is being created\n\n", formatter.Colorize(namespaceName, formatter.GREEN_COLOR), formatter.Colorize(ClusterName, formatter.GREEN_COLOR))
 
-		pitrConfigCtx := formatter.Context{
-			Output: os.Stdout,
-			Format: formatter.NewPitrConfigFormat(viper.GetString("output")),
-		}
+		if viper.GetBool("wait") {
```
Comment (arishta-yb) [inline cmd/cluster/pitr-config/pitr_config_ops.go:156]: This code block is being repeated at multiple places, can we extract it into a helper function?

Comment (arishta-yb) [reply cmd/cluster/pitr-config/pitr_config_ops.go:156]: Something like this perhaps?
`func handleTaskCompletion(authApi *ybmAuthClient.AuthApi, clusterID, msg string, taskType ybmclient.TaskTypeEnum)`

File: cmd/cluster/pitr-config/pitr_config_ops.go [hunk 2]
```diff
@@ -165,13 +237,97 @@ var restorePitrConfigCmd = &cobra.Command{
...
+	},
+}
+
+var deletePitrConfigCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete PITR Config for a cluster",
+	Long:  "Delete PITR Config for a cluster in YugabyteDB Aeon",
+	PreRun: func(cmd *cobra.Command, args []string) {
+		viper.BindPFlag("force", cmd.Flags().Lookup("force"))
+		namespaceName, _ := cmd.Flags().GetString("namespace-name")
```
Comment (arishta-yb) [inline cmd/cluster/pitr-config/pitr_config_ops.go:263]: Can you please add a null check for this?

File: cmd/cluster/pitr-config/pitr_config_ops.go [hunk 3]
```diff
@@ -97,15 +149,35 @@ var createPitrConfigCmd = &cobra.Command{
...
-		fmt.Printf("\nSuccessfully created PITR configuration.\n\n")
+		msg := fmt.Sprintf("The PITR Configuration for namespace %s in cluster %s is being created\n\n", formatter.Colorize(namespaceName, formatter.GREEN_COLOR), formatter.Colorize(ClusterName, formatter.GREEN_COLOR))
 
-		pitrConfigCtx := formatter.Context{
-			Output: os.Stdout,
-			Format: formatter.NewPitrConfigFormat(viper.GetString("output")),
-		}
+		if viper.GetBool("wait") {
+			returnStatus, err := authApi.WaitForTaskCompletion(clusterID, ybmclient.ENTITYTYPEENUM_CLUSTER, ybmclient.TASKTYPEENUM_ENABLE_DB_PITR, []string{"FAILED", "SUCCEEDED"}, msg)
+			if err != nil {
```
Comment (arishta-yb) [inline cmd/cluster/pitr-config/pitr_config_ops.go:158]: The error messages could be made more consistent. For example, in this command, you use `logrus.Fatalf("error when getting task status: %s", err)`, but in the delete command, it's `logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))`.

Comment (harshitha-yb) [reply cmd/cluster/pitr-config/pitr_config_ops.go]: The second error that you mentioned is from the create/delete/restore api calls which is consistent. The string error is from listTasks API calls

**Review by arishta-yb — APPROVED** [review]
LGTM!

**Pull request merged**